demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -1968,12 +1969,60 @@ def validate_job(self, structure_validator, pack_error_ignore_list):
 
         return is_valid
 
+    def get_all_files_edited_in_pack_ignore(self, modified_files):
+        all_files = self.git_util.get_all_files()
+        all_files_edited_in_pack_ignore = set()
+        for file_path in modified_files:
+            # handle renamed files
+            if isinstance(file_path, tuple):
+                old_file_path = file_path[0]
+                file_path = file_path[1]
+
+            else:
+                old_file_path = file_path
```

Comment:
is this idiom used elsewhere? I dont think its very intuitive. Can we make this a function, just for documentations sake?

Reply:
It's used in several places around the **validate_manager**.
I'll export it to an external function.